### PR TITLE
feat(decorator): Added support for extracting decorators from model files

### LINKED ISSUE #43: Add Support for extracting decorators and vocabularies from a list of models
add cli command for extraction of vocabularies and dcs from a list of models.

## Context
I wanted to extract decorators and vocabularies from model files

### Expected Behaviour
When I use CLI commands and provide  a data-model .cto file it should be able to extract and provide the new data-model along with decorators and vocabularies.

### PR DESCRIPTION
**Pull request by muskanbararia**
# Closes #43 
Added support to extract decorators and vocabularies from model files

### Author Checklist
- [x] Ensure you provide a [DCO sign-off](https://github.com/probot/dco#how-it-works) for your commits using the `--signoff` option of git commit.
- [x] Vital features and changes captured in unit and/or integration tests
- [x] Commits messages follow [AP format](https://github.com/accordproject/techdocs/blob/master/DEVELOPERS.md#commit-message-format)
- [ ] Extend the documentation, if necessary
- [x] Merging to `main` from `fork:branchname`

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -456,6 +456,42 @@ require('yargs')
...
+            alias: 'model',
+            type: 'string',
+            array:true,
+        });
+        yargs.option('locale', {
+            describe: 'The locale for extracted vocabularies',
+            type: 'string',
+            default:'en'
+        });
+        yargs.option('removeDecoratorsFromSource', {
```
Comment (mttrbrts) [inline index.js:472]: Does this option have any effect?

File: index.js [hunk 2]
```diff
@@ -456,6 +456,42 @@ require('yargs')
...
+        });
+    }, argv => {
+        const options = {
+            locale: argv.locale,
+            removeDecoratorsFromModel: argv.removeDecoratorsFromSource,
+            output: argv.output
+        };
+        return Commands.extractDecorators(argv.models,options)
+            .then(result => {
+                Logger.info(result);
```
Comment (mttrbrts) [inline index.js:489]: Do we always want to log the result? Even if we output to disk?

Also, what's the output format? A stringified JSON?

Comment (muskanbararia) [reply index.js:489]: It is a JSON currently. 
shall I include another option, for example `--console`, to determine whether to log or not?
We can keep one of `--output` and `--console` as mandatory.

File: index.js (hunk 1 above)
Comment (muskanbararia) [reply index.js:472]: Its a flag used to determine whether to remove the decorators from source model.

File: index.js (hunk 2 above)
Comment (mttrbrts) [reply index.js:489]: Given that there are multiple outputs to this command, I don't think that it makes sense to output content to the console. This is the same behaviour that we have with the compile command for example.

File: index.js (hunk 1 above)
Comment (mttrbrts) [reply index.js:472]: Sorry, I didn't see how it was passed into the Extractor.

File: index.js (hunk 2 above)
Comment (muskanbararia) [reply index.js:490]: Updated the PR

**Review by mttrbrts — APPROVED** [review]

**Pull request merged**